[Add] Implementation in Python

shin-k-h opened the pull request:
I wrote it because there was a demand for implementation in python.

shin-k-h committed:
add python link > README

shin-k-h committed:
[add] Implementation of written in Python

shin-k-h commented:
Is it better to marge CableBehavior class into Cable class?

k5342 commented on python/main.py:
```diff
@@ -0,0 +1,56 @@
+        return self.ohm
+
+class SerialCable(Cable):
+    def __init__(self, c):
```
I think it is preferable for defining a default value as `[]`. And the variable name `c` makes a little bit confused.

k5342 commented on python/main.py:
```diff
@@ -0,0 +1,56 @@
+    def getResistance(self):
+        return self.ohm
+
+class SerialCable(Cable):
```
I want `Cable#append` method to modify an existing cable after definition.

k5342 commented on python/main.py:
```diff
@@ -0,0 +1,56 @@
+            sum += p.getResistance()
+        return sum
+
+class ParalleCable(Cable):
```
typo: Parallel

k5342 commented on python/main.py:
```diff
@@ -0,0 +1,56 @@
+# !/path/to/bin/of/python
```
It seems unnecessary for Python script. It's not a shell script.

k5342 commented on python/main.py:
```diff
@@ -0,0 +1,56 @@
+class PartsBehavior(metaclass=ABCMeta):
+    @abstractmethod
+    def getResistance(self):
+        pass
```
Use `raise NotImplementedError` is preferable on an abstract class rather than `pass`.

k5342 reviewed (changes requested):
Thanks for sending PR. I wish some modify to merge this.

shin-k-h committed:
add comment into main.py

erased unnecessary sentences

rename c -> cables

fix typo

fix variable name in SerialCable

define SerialCable#append method

fixed for abstructmethod in PartsBehavior

add comments

fix README-py.md

shin-k-h commented:
修正しました

k5342 commented:
LGTM! ありがとうございます！

k5342 commented:
Revert しました #8 すみません．こちらひとつのコミットに squash して再度 PR 投げてもらえますか？

shin-k-h commented:
了解
